extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -169,6 +177,18 @@ void processAnnotationsAndIndexFiles(
                     proxyDefinitions.produce(new NativeImageProxyDefinitionBuildItem(className, Locatable.class.getName()));
                     addReflectiveClass(reflectiveClass, true, false, className);
                 });
+        produceProxyIfExist(proxyDefinitions, "com.sun.xml.bind.marshaller.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "com.sun.xml.internal.bind.marshaller.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "org.glassfish.jaxb.core.marshaller.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "com.sun.xml.txw2.output.CharacterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "org.glassfish.jaxb.characterEscapeHandler");
+        produceProxyIfExist(proxyDefinitions, "org.glassfish.jaxb.marshaller.CharacterEscapeHandler");
```

Comment:
Why ifExist? If the JAXB extension is there, don't we have them? Or are they different on various versions of the JDK?

Reply:
Package has moved between Java 8 and 13

Reply:
@dufoli can you check this is still needed since Quarkus 2.x now requires JDK 11 as a minimum?

Reply:
to be more precise it depend of version of jaxb.
jaxb 3.0 use org.glassfish.jaxb.core.marshaller.CharacterEscapeHandler
before it was on com.sun.xml.txw2.output.CharacterEscapeHandler

here is the commit:
https://github.com/eclipse-ee4j/jaxb-ri/commit/e863464a3a1d06a7955a90bb4bba4222f0cc63b6#diff-e5024e3cbb151f1d566536f1f1b29c2553073b623d3586d97948fcb715461728

it is detail in release note : 
https://github.com/eclipse-ee4j/jaxb-ri/releases

I see that quarkus still use 2.3.3-b02 version which is on old package format .
2 solutions:
- support old package path only
- switch to new package version 3.0 and support new package path only

what do you prefer ?

Reply:
@gastaldi

Reply:
I don't know what is the impact of bumping to version 3.0, maybe it's worth doing it and see if there are any failed tests